app/scripts/modules/core/src/managed/StatusBubble.tsx

```diff
@@ -9,6 +10,7 @@ export interface IStatusBubbleProps {
   iconName: IconNames;
   appearance: 'inactive' | 'neutral' | 'info' | 'progress' | 'success' | 'warning' | 'error';
   size: 'extraSmall' | 'small' | 'medium' | 'large' | 'extraLarge';
+  quantity?: string;
```

Comment:
does it make sense to allow `number`s here too? (i.e. if you're doing something like `someArray.length`)

Reply:
Good question. I've always thought that this would only come into picture during the `actuating` step where we are displaying the `finished/total` string. Is that not the case?

Reply:
🤷 medium term it'd be good to graduate this component into `core/presentation`, as it's going to be widely used across spinnaker. might not be worth doing it in this PR, but i bet we'll end up wanting to stick plain numbers on here (i'm already thinking about doing this with my pinning work)